sdks/python/apache_beam/io/gcp/bigquery_write_it_test.py

```diff
@@ -412,11 +420,103 @@ def test_big_query_write_temp_table_append_schema_update(self):
           | 'write' >> beam.io.WriteToBigQuery(
               table_id,
               schema=table_schema,
-              write_disposition=beam.io.BigQueryDisposition.WRITE_APPEND,
+              write_disposition=BigQueryDisposition.WRITE_APPEND,
               max_file_size=1,  # bytes
               method=beam.io.WriteToBigQuery.Method.FILE_LOADS,
               additional_bq_parameters={
-                  'schemaUpdateOptions': ['ALLOW_FIELD_ADDITION']}))
+                  'schemaUpdateOptions': ['ALLOW_FIELD_ADDITION']},
+              temp_file_format=file_format))
+
+  @pytest.mark.it_postcommit
+  @parameterized.expand([
+      param(file_format=FileFormat.AVRO),
+      param(file_format=FileFormat.JSON),
+      param(file_format=None),
+  ])
+  @mock.patch(
+      "apache_beam.io.gcp.bigquery_file_loads._DEFAULT_MAX_FILE_SIZE", new=1)
+  @mock.patch(
+      "apache_beam.io.gcp.bigquery_file_loads._MAXIMUM_SOURCE_URIS", new=1)
+  def test_append_schema_change_with_temporary_tables(self, file_format):
```

Comment:
What is this testing that `test_big_query_write_temp_table_append_schema_update` is not testing? Is it possible to do everything in one test?

Reply:
The first tests schema addition, the second - field relaxation. Yep, I can combine those two tests together